### PR TITLE
Adding Zalenium (Containerised Selenium Grid)

### PR DESCRIPTION
**Pull request by syvanen**
#### What is this PR About?
This adds Zalenium deployment. Current Selenium Grid example under [container-pipelines](https://github.com/redhat-cop/container-pipelines/tree/master/cucumber-selenium-grid) uses privileged ClusterRoles that are not always available. Also the upstream [Zelenium](https://opensource.zalando.com/zalenium/) uses Helm Charts that we currently don't have always available.

Once this PR is merged I will update the Selenium Grid example on container-pipelines.

#### How do we test this?
Follow instructions on README.md how to deploy and run simple test from your local machine.

cc: @redhat-cop/containerize-it

### CODE REVIEW

File: zalenium/.openshift/templates/zalenium-deployment.yml [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+                value: ${RETENTION_PERIOD}
+              - name: CONTEXT_PATH
+                value: ${CONTEXT_PATH}
+            resources:
+              request:
+                cpu: 500m
+                memory: "500Mi"
+              limits:
+                cpu: "1000m"
+                memory: "2Gi"
```
Comment (springdo) [inline zalenium/.openshift/templates/zalenium-deployment.yml]: I guess these memory limits should come from the template params too?

File: zalenium/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+Run the openshift-applier to create the `zalenium` project and deploy required objects
+```
+ansible-playbook -i .applier galaxy/openshift-applier/playbooks/openshift-cluster-seed.yml
+```
+
+### Next steps
+
+Access your Zalenium:
+* Life Dashboard of ongoing tests:
```
Comment (springdo) [inline zalenium/README.md]: s/Life/Live/g

**Comment by syvanen** [comment]
@springdo Added the improvements. Will start work on the labs-ci-cd and container-pipelines examples next.

**Comment by springdo** [comment]
Cool - As an optimistic merger 'm happy to slot this in!

**Pull request merged**